### ISSUE TITLE
Minimize to tray not working at start since 9.3-2.2 on Fedora 38

### ISSUE THREAD

**Issue by undeference** [Ximi1970/systray-x#152]
**Describe the bug**
I have it set to Minimize to tray, but minimizing Thunderbird shows it minimized in the window list. If I open systray-x-minimal's Preferences and then click Save (without even changing anything), then minimizing works properly.

The following Fedora 38 packages are affected:
systray-x-minimal-0.9.4-2.1.x86_64
systray-x-minimal-0.9.4-36.2.x86_64
systray-x-minimal-0.9.5-39.1.x86_64

The last package I know to work properly is
systray-x-minimal-0.9.3-2.2.x86_64

**Desktop (please complete the following information):**
 - OS: Fedora 38
 - Architecture 64 bit
 - Desktop type / Window Manager MATE
 - Thunderbird version and arch thunderbird-102.13.0-1.fc38.x86_64
 - Run `echo $XDG_SESSION_TYPE` in a terminal and report the results x11
 - Add-on version Version 0.9.2
 - App version (see above)
 - Install type: openSUSE repository for Fedora 38
 - Used Qt library qt-4.8.7-73.fc38.x86_64

**Comment by Ximi1970** [comment]
Installed also the Fedora 38 Mate edition as VM. Everything is working just fine with the 0.9.5 version. Maybe the preferences got lost? Please check the "Start application minimized" option.

**Comment by undeference** [comment]
I retested with systray-x-minimal-0.9.5-39.1.x86_64

> minimizing Thunderbird shows it minimized in the window list.

I should have said clicking the tray icon just minimizes Thunderbird normally. Both the minimize button and the window list entry minimize to tray, but clicking the tray icon does not (unless I open the Preferences and click Save first, but I have to do that every time I start Thunderbird).

> Installed also the Fedora 38 Mate edition as VM. Everything is working just fine with the 0.9.5 version. Maybe the preferences got lost?

The preferences all display as I had them (and they continue to work when downgrading).

> Please check the "Start application minimized" option.

What does Start minimized have to do with it?
I tried it and it properly starts minimized to the tray. If I restore the window and click the tray icon, it minimizes normally, not to the tray.

> Or try the "Reset" preferences but then you need to set all your preferences again.

I tried resetting my preferences and get the same result.
